README.md

```diff
@@ -1,189 +1,157 @@
-# c2paTool
+# c2patool - C2PA command line tool
 
-Command line tool for displaying and adding C2PA manifests  
-A file path to a JPEG or a claim definition JSON file must be provided  
-If a JPEG path is given, this will generate a summary report of any manifests in that file  
-If a manifest definition JSON file is specified, the manifest will be created and displayed in a JSON report
+c2patool is a command line tool for working with C2PA [manifests](https://c2pa.org/specifications/specifications/1.0/specs/C2PA_Specification.html#_manifests). Currently, the tool supports:
 
-## Displaying Manifest data
+- Reading a JSON report of C2PA manifests in [supported file formats](#supported-file-formats)
+- Reading a low-level report of C2PA manifest data in [supported file formats](#supported-file-formats)
+- Previewing manifest data from a [manifest definition](#manifest-definition-format)
+- Adding a C2PA manifest to [supported file formats](#supported-file-formats)
 
-Invoking the tool with a path to an image file will output a JSON report of the Manifests in the file
-File formats supported are jpeg and png. 
+## Supported file formats
 
-```c2patool image.jpg```
+- `image/jpeg`
+- `image/png`
 
-## Displaying detailed Manifest data
+## Installation
 
-The -d option will output a detailed JSON report of the internal C2PA structure
+### via Homebrew
 
-```c2patool image.jpg  -d```
+You can install c2patool on Mac and Linux via [Homebrew](https://brew.sh/):
 
-## Previewing a Manifest
+```shell
+brew tap contentauth/tools
+brew install c2patool
+```
+
+### Building from source
+
+If you have Rust installed, you can build c2patool from source:
+
+```shell
+git clone git@github.com:contentauth/c2patool.git
+cargo build
+```
+
+## Optional prerequisites
+
+### Manifest definition format
```

Comment:
It is more than a manifest definition now, it also contains configuration info for certs and such, so I've been referring to it as config file, and if you pass it on the command line, you use a -c config option.